lib/cloudapi2.js

```diff
@@ -3217,8 +3219,10 @@ CloudApi.prototype.createVolume = function createVolume(options, cb) {
         method: 'POST',
         path: format('/%s/volumes', this.account),
         data: {
+            affinity: options.affinity,
             name: options.name,
             size: options.size,
+            tags: options.tags,
             networks: (options.networks ? options.networks : undefined),
```

Comment:
If you tried to use alphabetical order tags should go after networks. If not, just ignore this comment :-)

Reply:
They are ordered alphabetical - just networks is (and always was) the odd sheep at the end (that I didn't want to touch).